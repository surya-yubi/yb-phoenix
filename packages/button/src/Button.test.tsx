import React from "react";
import renderer from "react-test-renderer";

import MyButton from "./index";

test("renders correctly", () => {
  const tree = renderer
    .create(<MyButton text={"Test Button"} onPress={() => console.log("")} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
