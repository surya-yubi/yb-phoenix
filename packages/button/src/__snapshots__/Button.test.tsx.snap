// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessibilityState={
    {
      "busy": undefined,
      "checked": undefined,
      "disabled": undefined,
      "expanded": undefined,
      "selected": undefined,
    }
  }
  accessibilityValue={
    {
      "max": undefined,
      "min": undefined,
      "now": undefined,
      "text": undefined,
    }
  }
  accessible={true}
  collapsable={false}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    {
      "backgroundColor": "violet",
      "opacity": 1,
      "paddingHorizontal": 16,
      "paddingVertical": 8,
    }
  }
>
  <Text
    style={
      {
        "color": "black",
      }
    }
  >
    Test Button
  </Text>
</View>
`;
